src/main/java/frc/robot/subsystems/RollerSubsystem.java

```diff
@@ -104,14 +122,23 @@ public void setRollPower(double power) {
     public void periodic() {
         // If `OPEN_TIME` hasn't elapsed yet, run the motor.
         boolean opening = openTimer.get() > 0 && !openTimer.hasElapsed(OPEN_TIME_SECONDS);
-        openMotor.set(opening ? 0.3 : 0);
+        openMotor.set(opening && allowOpen ? 0.5 : 0);
 
         // If the cooldown has passed, stop and reset the timer.
         if (openTimer.hasElapsed(OPEN_TIME_SECONDS + COOLDOWN_SECONDS)) {
             openTimer.stop();
             openTimer.reset();
         }
 
+        if(rolling){
```

Comment:
The `rolling` state seems unnecessary; why not check if the timer has started with `.get() > 0` like the open motor logic does?